components/filter/filter.js

```diff
@@ -142,8 +143,11 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 				color: var(--d2l-color-chromite);
 			}
 
+			.d2l-empty-state-container {
+				padding: 0.9rem;
```

Comment:
Needed to use padding in a container rather than margin to avoid margin collapse